OptimizelySDK/ForcedDecisionsStore.cs

```diff
@@ -0,0 +1,80 @@
+﻿/*
+ * Copyright 2021, Optimizely
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+using System.Collections.Generic;
+
+namespace OptimizelySDK
+{
+    /// <summary>
+    /// ForcedDecisionsStore defines helper methods that are used for optimizelyDecisionContext object manipulation.
+    /// </summary>
+    public class ForcedDecisionsStore
```

Comment:
Unit tests are already covered for this class as transitive dependency. But I will appreciate @mnoman09 to write unit tests.